rfcs/models/infrastructure/relays.md

```diff
@@ -21,11 +21,10 @@
 
 ## Relays for Cross-Ledger Communication
 
-Relays are the fulcra (or mediums) of communication across networks maintaining distributed ledgers. They perform the following functions:
+Relays are the fulcra (or media) of communication across networks maintaining distributed ledgers. They perform the following functions:
 - Pass messages conveying queries, instructions, and information from one network to another.
 - Mediate end-to-end protocols corresponding to [basic interoperability modes](../ledger/cross-ledger-operations.md) supported by Weaver.
 - Discover foreign networks' relays.
-- Route messages to foregn networks' relays if direct inter-relay communication is not possible.
```

Comment:
Why did you remove this? I know we don't support such routing right now but it's part of the vision (like discovery). Also, "foregn" is misspelled (should be "foreign").